ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/directionsquestionnaire/CCDDirectionsQuestionnaire.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.cmc.ccd.domain.directionsquestionnaire;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDAddress;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Builder
+@Value
+public class CCDDirectionsQuestionnaire {
+
+    private String languageInterpreted;
+    private String signLanguageInterpreted;
```

Comment:
Sign language interpreted should be YES or NO. I dont see it being string.

Reply:
This is free text input in front-end. The name is misleading here.